payload/Library/Scripts/install_or_defer.sh

```diff
@@ -332,6 +332,17 @@ if [[ $? -ne 0 ]]; then
     BAILOUT=true
 fi
 
+# Check if a custom CatalogURL is set and if it is available
+SUCatalog=$(python -c 'from Foundation import CFPreferencesCopyAppValue; print CFPreferencesCopyAppValue("CatalogURL", "com.apple.SoftwareUpdate")')
+if [ "$SUCatalog" != "None" ]; then
+  KernelVersion=$(uname -r)
+  curl --user-agent "Darwin/$KernelVersion" -s --head --request GET "$SUCatalog" | grep "200 OK" > /dev/null
```

Comment:
I don't think it's necessary to change from the default `curl` user-agent. As far as I know Apple is not doing agent filtering.

Reply:
Also I don't think `--request GET` is necessary. From the `curl` man page:

> Normally you don't need this option. All sorts of GET, HEAD, POST and PUT requests are rather invoked by using dedicated command line options.

Reply:
> I don't think it's necessary to change from the default `curl` user-agent. As far as I know Apple is not doing agent filtering.

The user-agent may be required for Reposado users doing server-side URL-rewrites based on it.

> Also I don't think --request GET is necessary.

Yep, you're absolutely right.